src/web/screens/manageParticipants.tsx

```diff
@@ -46,7 +47,8 @@ function ManageParticipants() {
   const navigate = useNavigate();
 
   useEffect(() => {
-    if (!LoggedInUser?.user?.isUid2Support && participant) {
+    const user = LoggedInUser?.user as UserWithIsUid2Support;
+    if (user?.isUid2Support && participant) {
```

Comment:
I believe the first condition is reversed, it should be: `if (!user?.isUid2Support && participant) {`